packages/curriculum-parser-json/index.js

```diff
@@ -0,0 +1,130 @@
+const {
+  getParser: getMarkdownParser,
+} = require('@enkidevs/curriculum-parser-markdown')
+const { sectionNames, contentTypes } = require('@enkidevs/curriculum-helpers')
+const unistBuilder = require('unist-builder')
+const unistAssert = require('unist-util-assert')
+const { safeDump } = require('js-yaml')
+
+function getParser(type) {
+  function parse(json) {
+    return Promise.resolve(parseSync(json))
+  }
+
+  function parseSync(json) {
+    switch (type) {
+      case contentTypes.EXERCISE:
+      case contentTypes.INSIGHT:
+        return insightParser(json)
+      default:
+        throw new Error(`Invalid parser type [${type}] requested`)
```

Comment:
We should probably say `"Unsupported"` instead of `"Invalid"` because we will support all content nodes eventually